### PR TITLE
Update Creating A Blog tutorial to v4.0

### PR DESCRIPTION
**Pull request by hungrybluedev**
- Fixes the link so that GitBook actually generates the blog tutorial
- Changes the content to reflect v4.0's API

### CODE REVIEW

File: prologue/create-a-blog.md [hunk 1]
```diff
@@ -143,6 +143,16 @@ You'll notice now that we are returning the `blog` view but it does not exist ye
 
 All views are in the `templates` directory. We can create a new file called `templates/blog.html` .
 
+This can be done with the `touch` command on Unix systems:
+
+{% tabs %}
+{% tab title="terminal" %}
+```text
+$ touch templates/blog.html
```
Comment (josephmancuso) [inline prologue/create-a-blog.md]: This won't work on windows right?

**Review by josephmancuso — CHANGES_REQUESTED** [review]

File: prologue/create-a-blog.md (hunk 1 above)
Comment (hungrybluedev) [reply prologue/create-a-blog.md]: Updated the passage to be more general.

**Review by Marlysson — CHANGES_REQUESTED** [review]
I think there's a misspelling word.

> Also, feel free to change the ~~`DB_DATABSE`~~ `DB_DATABASE` name to something else.

**Review by josephmancuso — CHANGES_REQUESTED** [review]
https://github.com/MasoniteFramework/docs/blob/1b98eebf3822d17ac778a0029d997b18b70896bd/prologue/create-a-blog.md?plain=1#L79

This should be `app/controllers`

**Comment by josephmancuso** [comment]
Made some changes

**Review by josephmancuso — APPROVED** [review]
Looks good to me 👍

**Pull request merged**